zeppelin-web-angular/src/app/pages/workspace/notebook/paragraph/control/control.component.ts

```diff
@@ -193,6 +195,14 @@ export class NotebookParagraphControlComponent implements OnInit, OnChanges {
         icon: 'api',
         trigger: () => this.toggleEnabled(),
         shortCut: `R (Command)`
+      },
+      {
+        label: 'Debug',
+        show: this.defaultInterpreterGroup === 'jdbc' ? true : false,
```

Comment:
@zhugezifang In my understanding, `defaultInterpreterGroup` can be changed dynamically if you create a new interpreter from the type of interpreter. E.g. We can have `jdbc2` as a `jdbc` type but it won't work properly. So that's the reason why I don't think we can control the type of interpreters. Moreover, we can change the type of paragraph dynamically from jdbc to spark and flink. In this case will it work properly? The third, I don't know if this debug feature is general for all kinds of jdbc-related engine. We can use several types of jdbc drivers by default but I don't know how we can guarantee current jdbc provide jdbc features.

In another aspect, I feel like adding a new interpreter seems quite good. Do you have any concerns to add a new interpreter in the jdbc interpreter group?